modules/Library.js

```diff
@@ -0,0 +1,38 @@
+import { booksContainer } from './queries.js';
+
+export default class Library {
+  constructor() {
+    this.bookList = [];
+    const localData = localStorage.getItem('bookList');
+    if (localData !== null) {
+      this.bookList = JSON.parse(localData);
+    }
```

Comment:
- [ ] Hi Nice work. As part of the requirements please kindly change the library class and methods to an arrow functions